ui/mock/triggerStream.ts

```diff
@@ -0,0 +1,56 @@
+import { FunctionRunStatus, FunctionTriggerTypes } from "../src/store/generated";
+
+export const triggerStream = [
+    {
+      id: 'id1',
+      name: 'payment.intent.created',
+      type: FunctionTriggerTypes.Event,
+      startedAt: '2023-07-17T00:08:42.725+01:00',
+      source: {
+        type: 'webhook',
+        name: 'Stripe',
+      },
+      test: true,
+      functions: [
+        {
+          id: 'function1',
+          name: 'Email: Make Payment',
+          status: FunctionRunStatus.Completed,
+        },
+        {
+          id: 'function2',
+          name: 'Discord: Send Notification',
+          status: FunctionRunStatus.Running,
+        },
+      ],
```

Comment:
**question (non-blocking)**: Is this a list of functions or function runs? If we were to add links to each would it link to the function or the function run?

Reply:
It is function runs, I changed the name so it's more accurate!